dev-docs/source/Testing/SANSGUI/ISISSANSGUITests.rst

```diff
@@ -194,6 +194,24 @@ Processing
 #. Check that your save location contains files for both the background subtracted workspace and the normal reduction
    output.
 
+Save Other
+##########
+
+*Single Workspace*
+
+#. On the ``Runs`` tab, with some workspaces present in the ADS, click the ``Save Other`` button.
```

Comment:
Just to avoid any potential for confusion, is it worth saying "with some reduced workspaces present"?